tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -2053,6 +2052,7 @@ public function userHasCreatedPublicLinkToEntityInsideOfSpaceRequest(
 
 	/**
 	 * @Given /^user "([^"]*)" has shared a space "([^"]*)" to user "([^"]*)" with role "([^"]*)"$/
+	 * @When /^user "([^"]*)" updates the space "([^"]*)" for user "([^"]*)" changing the role to "([^"]*)"$/
```

Comment:
@kiranparajuli589 did you mean this in your previous review https://github.com/owncloud/ocis/pull/5088#discussion_r1028841057?

Reply:
yes :+1: